instrumentation/netty-4.0/src/main/java/io/opentelemetry/auto/instrumentation/netty40/ChannelFutureListenerInstrumentation.java

```diff
@@ -84,25 +82,27 @@ public static AgentScope activateScope(@Advice.Argument(0) final ChannelFuture f
       if (cause == null) {
         return null;
       }
-      final AgentSpan parentSpan =
+      final Span parentSpan =
           future.channel().attr(AttributeKeys.PARENT_CONNECT_SPAN_ATTRIBUTE_KEY).getAndRemove();
       if (parentSpan == null) {
         return null;
       }
-      final AgentScope parentScope = activateSpan(parentSpan, false);
+      final Scope parentScope = NettyHttpServerDecorator.TRACER.withSpan(parentSpan);
```

Comment:
static import?

Reply:
i was following similar import for DECORATE below, i was thinking maybe this was intentional here since it was not obvious in this file which would be used (NettyHttpServerDecorator or NettyHttpClientDecorator)